pkg/client/spaceblobadd.go

```diff
@@ -220,7 +219,7 @@ func (c *Client) SpaceBlobAdd(ctx context.Context, content io.Reader, space did.
 				return AddedBlob{}, fmt.Errorf("unexpected capability in accept task: %s", ability)
 			}
 		default:
-			log.Warnf("ignoring receipt for unexpected task: %s", concludeRcpt.Ran().Link())
+			log.Infof("ignoring receipt for unexpected task: %s", concludeRcpt.Ran().Link())
```

Comment:
Hmm, are you seeing this? What is that invocation? Can we warn if we don't know what it is but ignore/not log if we do?